docs/security/users-and-teams/auditing.md

```diff
@@ -20,6 +24,7 @@ Some  general points worth noting:
 - Octopus **does** capture the details of every mutating action (create/edit/delete) including who initiated the action.
 - Octopus **does not** capture login and logout events for specific user accounts.
 - Octopus **does not** capture when data is read, however certain sensitive actions like downloading a certificate with its private key is captured.
+- Entries older than 90 days (default, configurable up to 365 days) are archived and deleted from the database. As such they are not queryable through the user interface. 
```

Comment:
```suggestion
- From **Octopus 2022.3** and later, entries older than 90 days (default, configurable up to 365 days) are archived and deleted from the database. As such, they are not queryable through the user interface. 
```

Reply:
This is only a suggestion. Not sure how it works with any feature flag usage for self-hosted customers

Reply:
@harrisonmeister we mention a version at the top as well as the fact that not all customers will see that. So I think it is sufficient and we shouldn't repeat it again.

Reply:
@harrisonmeister as @irynasemenova mentioned, we do list the version at the top and as you noted, with feature toggles it's going to be difficult to predict exactly when on-prem are going to see this. I'm comfortable with the callout at the top of the dock. 

@sburmanoctopus @irynasemenova this did get me thinking about the wording here. I wonder if we could tweak it to something like the follow, just to be explicit -
`Entries older than 90 days (default, configurable up to 365 days) are archived and deleted from the database. Archived events are not queryable through the user interface.`

Reply:
Sounds good. Done!

Reply:
Works for me :+1: